apps/web/src/styles.css

```
@@ -1 +1,6 @@
 @import '@altstack/ui/globals.css';
+@source "../node_modules/streamdown/dist/*.js";
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)